src/engine/TPA.cc

```diff
@@ -1305,10 +1305,14 @@ class TransitionSystemNetworkManager {
         std::unique_ptr<TPABase> solver {nullptr};
         PTRef trulyReached {PTRef_Undef};
         PTRef trulySafe {PTRef_Undef};
-        EId next;
-        EId previous;
+        PTRef accumulatedRestrictions;
+        vec<EId> children;
+        int blocked_children;
+        vec<EId> parents;
```

Comment:
I believe you do not need to store the parents. We are backtracking using the active path and that is enough.

Reply:
The thing is, it will allow to possibly update our implementation, for example to parse the graph not from the beginning to the end, but in the reverse order. I can remove it of course, but it is a placer for a possible system updates, wdyt?